force_wfmanager/central_pane/plot.py

```diff
@@ -43,6 +63,34 @@ class Plot(HasStrictTraits):
     #: Listens to: :attr:`x`, :attr:`y`
     _plot = Instance(Component)
 
+    #: Possible colormaps given color_map type
+    #: Listens to :attr:`colormap_type` and relies on default being the
+    #: The default is set by the first entry of this list.
+    __continuous_colormaps = Dict(
+        {cmap.__str__().split()[1]: cmap
+         for cmap in color_map_functions}
+    )
+    __discrete_colormaps = Dict(
```

Comment:
I think you did mention it already, but what was the issue with the discrete maps? Was it that they crashed for non int values?
If it seemed like it would be quite a difficult thing to fix, I'd remove the `__discrete_colormaps` and associated attrs for now. But if not we can leave it in and wait for the fix before merging this

Reply:
Basically, yeah. If you try to use a discrete colourmap for floats, chaco trashes the plot. One possible way around this would be to mark certain columns in the analysis model as "categorical", but it might be more effort than its worth.